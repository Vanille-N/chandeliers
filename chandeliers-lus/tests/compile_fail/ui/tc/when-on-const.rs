//! Can't use `when` on a constant.
chandeliers_lus::decl! {
    const X : int when b = 0;
}

fn main() {}
