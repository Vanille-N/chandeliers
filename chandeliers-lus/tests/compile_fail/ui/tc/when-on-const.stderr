error: expected `=`
 --> tests/compile_fail/ui/tc/when-on-const.rs:3:19
  |
3 |     const X : int when b = 0;
  |                   ^^^^
